tests/Cron/DaylightSavingsTest.php

```diff
@@ -342,7 +403,6 @@ public function testOffsetDecrementsEveryOtherHour(): void
         $expected = [
             $this->createDateTimeExactly("2020-10-24 22:00+01:00", $tz),
             $this->createDateTimeExactly("2020-10-25 00:00+01:00", $tz),
-            $this->createDateTimeExactly("2020-10-25 01:00+00:00", $tz),
```

Comment:
Didn't noticed this failing test locally due to running php8. Will look into how to properly resolve the failing test.